deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/JobsResponseParser.java

```diff
@@ -20,30 +20,29 @@
  */
 public class JobsResponseParser
 {
-    @Expose(deserialize = false)
-    private static final String DATEFORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'";
-    @Expose(deserialize = false)
-    private static final String TIMEZONE = "UTC";
```

Comment:
unused?

Reply:
Yep, unused. Even weirder is that they were exposed for json serialization